content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -537,6 +537,18 @@ spec:
       ...
 ```
 
+Alternatively, a job suspension can be toggled from the command line by patching the job resource.  Suspend an active job:
```

Comment:
I'm not sure how necessary the command-line steps are to add though. We don't currently give full instructions for updating the manifest.

Reply:
Thanks for your awesome suggestions above @shannonxtreme, I think this covers off this nicely 😊. Will make another commit following this discussion change one of the 'job' references to 'Job' for consistency.

As for the command line patch options, I'm happy to go either way on this.  The main objective on this PR from my side was to get the main message that we've already covered above.

I did however find myself looking for a convenient means of toggling the suspend functionality within the CLI, hence it's inclusion in the PR.

Personally, I use the CLI more often where there is a shortcut (akin to kubectl create) or, for reactive changes which arguably a job suspension/resume would be.  I do however understand your feedback and viewpoint regarding the docs providing instructions on updating manifests.

I'll leave it at your discretion with no regrets if it's removed from the PR, let me know and if needed I'll update the commit.

Kind Regards


James Spurin